rfcs/text/0000-Managed-Access-Data.md

```diff
@@ -0,0 +1,346 @@
+### **DCP PR:**
+
+*Leave this blank until the RFC is approved then the Author(s) must
+create a link between the assigned RFC number and this pull request in
+the format:*
+
+[dcp-community/rfc\#](https://github.com/HumanCellAtlas/dcp-community/pull/\<PR\#\>)
+
+**Informational RFC: Managed Access Data in the Datas Control Platform**
+========================================================================
+
+**Summary**
+-----------
+
+This RFC describes the requirements for the DCP to support storing and
+controlling access to managed access data.
+ 
+**Author(s)**
+-------------
+
+*Recommended format for Authors:*
+
+[Kevin
+Osborn]([[mailto:kosborn2\@ucsc.edu]](mailto:kosborn2@ucsc.edu))
+
+[Trent
+Smith]([[mailto:trent.smith\@chanzuckerberg.com]](mailto:trent.smith@chanzuckerberg.com))
+
+[Brian
+Hannafious]([[mailto:bhannafi\@ucsc.edu]](mailto:bhannafi@ucsc.edu))
+
+[Brian
+O'Connor]([[mailto:broconno\@ucsc.edu]](mailto:broconno@ucsc.edu))
+
+**Shepherd**
+------------
+
+[Name](mailto:username\@example.com)
+
+**Motivation**
+--------------
+
+Many researchers collecting cellular resolution genomic data are doing
+so using donor consents which require the sequence level and any
+clinical phenotypic data to only be released to bona fide researchers
+for specific research use. This can happen for many reasons including
+the national law where they are based or the types of individuals they
+are collecting data from e.g individuals with a disease. This means
+there is a growing set of data suitable for building the Human Cell
+Atlas (HCA) which can't currently be stored, processed and distributed
+by the Data Coordination Platform (DCP).
+
+The processing of genetic and genomic information necessarily raises
+questions as to what information is 'identified or identifiable' (and
+thus subject to GDPR) and what information is 'anonymous or anonymized'
```

Comment:
_GDPR_ -> _General Data Protection Regulation (GDPR)_ perhaps linked to https://eugdpr.org/.

Reply:
thanks